SConstruct

```diff
@@ -314,19 +315,20 @@ if SHARED:
 else:
   cereal = [File('#cereal/libcereal.a')]
   messaging = [File('#cereal/libmessaging.a')]
+  visionipc = [File('#cereal/libvisionipc.a')]
```

Comment:
why don't we import these from the cereal SConscript?

Reply:
Not sure. Decided to do it the same as messaging, but can look into it after this PR.